src/app/Plans/components/Wizard/mocks/VMs.mock.ts

```diff
@@ -0,0 +1,146 @@
+import { IVMs } from '../../../types';
+
+export const VMsOptions = [
```

Comment:
As mentioned above, instead of putting the tree view item structure in here, let's put mock data in here of datacenters and clusters and folders the way we expect them to be in the API. Here is an example you can look at in Jeff's demo API server: https://inventory-openshift-migration.apps.cluster-jortel.v2v.bos.redhat.com/namespaces/openshift-migration/providers/vsphere/test/tree/vm (requires VPN)

Reply:
Actually let's talk on Slack about this, Jeff is telling me that might not make sense:

> The datacenters/clusters/folders are not the same tree.
> it's either datacenter/clusters (by folder)
> or
> datacenter/folders/vms